### PR TITLE
Added build script for Windows to Unity docs

### PR DESCRIPTION
**Pull request by aperekalin**
Previously the build script only had items for Android and iOS. I think it's worth adding workflows for Windows there as well. Maybe there are other parts of the Codemagic Unity docs that we need to pay attention to, but the build script is obviously missing. Borrowed one from an upcoming article by Mina.

### CODE REVIEW

File: content/yaml-quick-start/building-a-unity-app.md [hunk 1]
```diff
@@ -189,6 +189,20 @@ public static class BuildScript
...
+        buildPlayerOptions.locationPathName = "win/" + Application.productName + ".exe";
+        buildPlayerOptions.target = BuildTarget.StandaloneWindows;
+        buildPlayerOptions.options = BuildOptions.None;
+        buildPlayerOptions.scenes = GetScenes();
+
+        Debug.Log("Building Windows");
+        BuildPipeline.BuildPlayer(buildPlayerOptions);
+        Debug.Log("Built Windows");
+    }
+
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-unity-app.md:205]: This page is a quick start for building 'Building Unity **mobile** apps' for Android and iOS. 

Not sure how adding the script for building a Windows desktop fits in here.

Comment (aperekalin) [reply content/yaml-quick-start/building-a-unity-app.md:205]: We don't have any docs for Unity & Windows. Even for the Steam publishing, where Windows is the main platform. If not here, then we need some other place for that.

Comment (nevercode-kevin) [reply content/yaml-quick-start/building-a-unity-app.md:205]: Let's put it here for now. We can possibly come up with a better solution at a later date.

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**